requirements.txt

```diff
@@ -1,5 +1,5 @@
 qiskit>=0.25
-git+https://github.com/PennyLaneAI/pennylane.git@master
+pennylane >= 0.17.0
```

Comment:
```suggestion
pennylane>=0.17
```

Reply:
Oh, likely the version of PennyLane needs to be changed in the `setup.py` file too to `pennylane>=0.17`